packages/mui-material/src/Popover/Popover.js

```diff
@@ -359,6 +361,24 @@ const Popover = React.forwardRef(function Popover(inProps, ref) {
   const container =
     containerProp || (anchorEl ? ownerDocument(resolveAnchorEl(anchorEl)).body : undefined);
 
+  const Paper = slots?.paper ?? PopoverPaper;
+
+  const paperSlotProps = slotProps?.paper ?? PaperProps;
```

Comment:
I think it would be helpful to document below (under `PaperProps`) that it will be overridden by `slotProps.paper` if both are provided